src/lib/common/pcap.c

```diff
@@ -105,6 +105,5 @@ void pcapEnable() {
 void pcapDisable() {
   if(pcapSerialHandle) {
     pcapSerialHandle->enabled = false;
-    printf("PCAP Stream disabled!\n");
```

Comment:
this causes a crash because `pcapDisable` is executed from the `vusbDetectIRQHandler`.